src/commands/ext-install.ts

```diff
@@ -114,17 +116,58 @@ async function installExtension(options: InstallExtensionOptions): Promise<void>
 /**
  * Command for installing a extension
  */
-export default new Command("ext:install <extensionName>")
-  .description("install an extension, given <extensionName> or <extensionName@versionNumber>")
+export default new Command("ext:install [extensionName]")
+  .description(
+    "install an extension, provide [extensionName] or [extensionName@versionNumber]; or omit and run with `-i` to see all available extensions."
```

Comment:
I don't like how this reads. I think it's also incorrect to document a flag in the description.

```suggestion
    "install an extension provided [extensionName] or [extensionName@versionNumber]"
```

"provided [extensionName[@versionNumber]]" is even more succinct, but that might be pushing it too much.

Reply:
Is this trying to say:
This command installs an extension if you provide the `extensionName` or the `extensionName@versionNumber` ?

Reply:
I tried to shorten the sentence and make it more clear. PTAL